more/problem-sets-competitive-programming.md

```diff
@@ -25,6 +25,7 @@
 * [Coderbyte](https://coderbyte.com)
 * [Codewars](http://www.codewars.com)
 * [Codingame](https://www.codingame.com/start)
+* [CodingNinjas](https://www.codingninjas.com/codestudio/problems)
```

Comment:
Patch to https://github.com/EbookFoundation/free-programming-books/pull/6073#issuecomment-938701189 suggestion

```suggestion
* [Coding Interview Questions and answers for practice \| Python, Java & C++](https://www.codingninjas.com/codestudio/problems) - CodingNinjas
```

Reply:
Resolved by https://github.com/EbookFoundation/free-programming-books/commit/015e4b0368f5ac52b21cb22facc736a9152ce930